### DISCUSSION TITLE
12.8 and 13.1 OCR releases don't compile for me

### DISCUSSION THREAD

**Discussion by RossieAC** [sanni/cartreader#891] [Need help]
Hi. Release 12.8 and 13.1 required me to add hardware version when compiling. I have added "#define HW5" I get another error "checkAltConf exit status 1 'checkButton' was not declared in this scope" I don't know how to fix that one, so I can't flash//try these newer versions to try to fix CRC32 problem I just posted separately. Are these versions working for everyone else?

**Comment by Ancyker** [comment]
Where did you add it?
